lib/src/components/NxNexusPageHeader/NxNexusPageHeader.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+const defaultLogo = require('../../assets/img/logo_nexus_generic.svg');
+
+import AbstractNxPageHeader from '../AbstractNxPageHeader/AbstractNxPageHeader';
+import { Props, ProductInfo, propTypes } from './types';
+
+export { Props };
+
+import './NxNexusPageHeader.scss';
+import { join, reject, isNil } from 'ramda';
+
+function HeaderProductInfo({ name, meta, version }: ProductInfo) {
+  // note that combining these in JSX with a space in between would result in an extra space when one of the
+  // fields isn't present, so we have to combine them more carefully like this
+  const metaAndVersionString = join(' ', reject(isNil, [meta, version]));
+
+  return (
+    <>
+      <div className="nx-product__summary">
+        <span className="nx-product__brand">nexus</span>
+        <span className="nx-product__name">{name}</span>
+      </div>
+      <div className="nx-product__details">
+        <span className="nx-product__byline">from sonatype</span>
+        { metaAndVersionString && <span className="nx-product__meta">{metaAndVersionString}</span> }
+      </div>
+    </>
+  );
+}
+
+export default function NxNexusPageHeader({ productInfo, logoPath, ...otherProps }: Props) {
+  const logo = <img src={logoPath || defaultLogo} className="nx-product__logo-image" />,
```

Comment:
I believe we need an empty `alt` attr here.  I just read through the guidelines for that attribute, and this part seemed most relevant:

> If the logo is being used next to the name of the entity that it represents, then the logo is supplemental, and its alt attribute must instead be empty.

Here's the entire set of guidelines: https://html.spec.whatwg.org/multipage/images.html#alt

Reply:
> I believe we need an empty `alt` attr here. I just read through the guidelines for that attribute, and this part seemed most relevant:
> 
> > If the logo is being used next to the name of the entity that it represents, then the logo is supplemental, and its alt attribute must instead be empty.
> 
> Here's the entire set of guidelines: https://html.spec.whatwg.org/multipage/images.html#alt

https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/28ca779b3e4e3541db5de1e34889dbbf508b50ce